modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequest.testcase

```diff
@@ -144,6 +144,74 @@ definition {
 		}
 	}
 
+	@description = "LPS-180356 - Verify that  the 'phone number' field is enabled  if it's not registered and it is possible to fill this"
+	@priority = 3
+	test IsPossibleFillPhoneNumberField {
+		task ("Given the Employee user don't have a phone number registered") {
```

Comment:
Just a typo here, change the task to : `Given the Employee user doesn't have a phone number registered`